config/opal_check_pc.m4

```diff
@@ -0,0 +1,40 @@
+dnl -*- shell-script -*-
+dnl
+dnl Copyright (c) 2021 IBM Corporation.  All rights reserved.
+dnl
+dnl $COPYRIGHT$
+dnl
+dnl Additional copyrights may follow
+dnl
+dnl $HEADER$
+dnl
+
+AC_DEFUN([OPAL_CHECK_LFLAGS], [
+         AC_MSG_CHECKING([for pc file for])
+         AC_MSG_RESULT([$1])
+
+	 num_paths=$#
+         save_name=$1
+         cmd="pkg-config"
+         $1_OMPI_PC_DIR=""
+         if test $num_paths -gt 1; then
```

Comment:
a comment about what you're doing here would be appreciated.

Reply:
Updated with comments.